docs/source/llms/tracing/tracing-schema.rst

```diff
@@ -0,0 +1,328 @@
+-----------
+
+Spans are the core of the trace data. They record key, critical data about each of the steps within your genai application. 
+
+When you view your traces within the MLflow UI, you're looking at a collection of spans, as shown below. 
+
+.. figure:: ../../_static/images/llms/tracing/schema/spans_in_mlflow_ui.png
+    :alt: Spans within the MLflow UI
+    :width: 100%
+    :align: center
+
+The sections below provide a detailed view of the structure of a span.
+
+Inputs
+^^^^^^
+
+The inputs are stored as JSON-serialized strings, representing the input data that is passed into the particular stage (step) of your application. 
+Due to the wide variety of input data that can be passed between specific stages of a GenAI application, this data may be extremely large (such as when 
+using the output of a vector store retrieval step). 
+
+Reviewing the Inputs, along with the Outputs, of individual stages can dramatically increase the ability to diagnose and debug issues that exist with responses 
+coming from your application.
+
+Outputs
+^^^^^^^
+
+The outputs are stored as JSON-serialized strings, representing the output data that is passed out of the particular stage (step) of your application. 
+Just as with the Inputs, the Outputs can be quite large, depending on the complexity of the data that is being passed between stages.
+
+Attributes
+^^^^^^^^^^
+
+Attributes are metadata that are associated with a given step within your application. These attributes are key-value pairs that can be used to provide insight 
+into behavioral modifications for function and method calls, giving insight into how modification of them can affect the performance of your application. 
+
+Some common examples of attributes include:
+
+- **model** - The name and version of a given external service when calling an LLM provider.
+- **temperature** - The amount of creativity you are configuring for the answer from an LLM when calling it. 
+- **document_count** - For vector store retrieval stages, the number of relevant documents to return to provide context to an LLM to answer a question.
+
+Events
+^^^^^^
+
+Events are a system-level property that is optionally applied to a span only if there was an issue during the execution of the span. These events contain 
+information about exceptions that were thrown in the instrumented call, as well as the stack trace. This data is structured within an 
+:py:class:`mlflow.entities.SpanEvent` object.
+
+SpanEvent
+~~~~~~~~~
+
+The :py:class:`Span Event <mlflow.entities.SpanEvent>` object, if present, consists of three elements:
+
+- **name** - defines the name of the event that occurred.
+- **timestamp** - the time at which the event occurred in microseconds.
+- **attributes** - a key-value collection of information related to the event that occurred. In the event of an exception being thrown, the stack trace 
+  will be included in this collection to aid in debugging and diagnosing the root cause of issues.
```

Comment:
```suggestion
```

nit: this seems to be repeating https://mlflow.org/docs/latest/python_api/mlflow.entities.html#mlflow.entities.SpanEvent (which is already linked)

Reply:
refactoring this